<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasternodeList</class>
 <widget class="QWidget" name="MasternodeList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="topLayout" stretch="0">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_Master" stretch="1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_BG">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QFrame" name="frame_Content">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="spacing">
              <number>2</number>
             </property>
             <item row="0" column="0">
              <widget class="QFrame" name="frameList">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <property name="bottomMargin">
                 <number>1</number>
                </property>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_note">
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="updateNote">
                    <property name="text">
                     <string>Note: Status of your masternodes in local wallet can potentially be slightly incorrect.&lt;br /&gt;Always wait for wallet to sync additional data and then double check from another node&lt;br /&gt;if your node should be running but you still see &quot;MISSING&quot; in &quot;Status&quot; field.</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QTableWidget" name="tableWidgetMyMasternodes">
                  <property name="minimumSize">
                   <size>
                    <width>695</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="palette">
                   <palette>
                    <active>
                     <colorrole role="Text">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>51</red>
                        <green>51</green>
                        <blue>51</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </active>
                    <inactive>
                     <colorrole role="Text">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>51</red>
                        <green>51</green>
                        <blue>51</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </inactive>
                    <disabled>
                     <colorrole role="Text">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>85</red>
                        <green>85</green>
                        <blue>85</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </disabled>
                   </palette>
                  </property>
                  <property name="editTriggers">
                   <set>QAbstractItemView::NoEditTriggers</set>
                  </property>
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                  <property name="selectionMode">
                   <enum>QAbstractItemView::SingleSelection</enum>
                  </property>
                  <property name="selectionBehavior">
                   <enum>QAbstractItemView::SelectRows</enum>
                  </property>
                  <property name="sortingEnabled">
                   <bool>true</bool>
                  </property>
                  <attribute name="horizontalHeaderStretchLastSection">
                   <bool>true</bool>
                  </attribute>
                  <column>
                   <property name="text">
                    <string>Alias</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Address</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Protocol</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Status</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Active</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Last Seen (UTC)</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Pubkey</string>
                   </property>
                  </column>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QtMaterialFlatButton" name="startButton" native="true">
                    <property name="text" stdset="0">
                     <string>Start alias</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QtMaterialFlatButton" name="startAllButton" native="true">
                    <property name="text" stdset="0">
                     <string>Start all</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QtMaterialFlatButton" name="startMissingButton" native="true">
                    <property name="text" stdset="0">
                     <string>Start MISSING</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QtMaterialFlatButton" name="UpdateButton" native="true">
                    <property name="text" stdset="0">
                     <string>Update status</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="autoupdate_label">
                    <property name="text">
                     <string>Status will be updated automatically in (sec):</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="secondsLabel">
                    <property name="text">
                     <string>0</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_5">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialFlatButton</class>
   <extends>QWidget</extends>
   <header>qtmaterialflatbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
